src/sass/components/_section--features.scss

```diff
@@ -131,21 +133,34 @@
     }
     .furniture {
       max-height: 490px;
-      background: url('../images/features/sofa.jpg');
-      background-size: cover;
-      background-position: center;
+      height: 490px;
       padding: 0;
-      &:hover {
-        .price {
-          .btn-main-small {
+      &:hover .price .btn-main-small {
             background-color: $primary;
-          }
-        }  
+      }  
+      .fur1 {
+        background: url('../images/features/sofa.jpg');
+      }
+      .fur2 {
+        background: url('../images/features/outdoor.jpeg');
+      }
+      .fur3 {
+        background: url('../images/features/decorations.jpeg');
       }
+      .carousel-inner {
```

Comment:
Tutaj ładnie można się w to zagnieździć :) 
```
.carousel {
    &-inner {}
    &-item {}
}
```

korzystajmy z tych rozwiązać, żeby ułatwiać sobie życie :)

Reply:
Te rozwiązania rzeczywiście są bardzo fajne, ale w takich nieoczywistych przypadkach jeszcze nie do końca weszły mi w krew. Dobra uwaga. :-)